GliaWidgets/Sources/View/Chat/Message/ChatMessageView.swift

```diff
@@ -63,10 +63,6 @@ class ChatMessageView: BaseView {
             )
             contentView.attributedText = text
             contentView.linkTapped = { [weak self] in self?.linkTapped?($0) }
-            contentView.accessibilityProperties = .init(
```

Comment:
Is there a reason this was removed?

Reply:
This implementation is applicable only to a normal message, and not to an attributed message. Removing that was essential for the accessibility to be displayed properly